package.json

```diff
@@ -11,7 +11,8 @@
     "watch": "cd ui && npm run watch && cd ..",
     "heroku-prebuild": "cd ui && mkdir build && npm install",
     "heroku-postbuild": "cd ui && npm run build",
-    "test": "cd ui && npm run test"
+    "lint": "eslint --ext jsx --ext js -c server/.eslintrc server",
```

Comment:
This is adding commands to run linting on the server code too, I thought that was already set up but wasn't.